odata-protocol/11.5 Operations.md

```diff
@@ -208,6 +208,10 @@ POST http://host/service/MyShoppingCart()/Items
 ```
 :::
 
+If the function returns a value of type `Edm.Stream` and no additional path
+segments follow the function invocation, the response to the `GET` request
+obeys the rules for [requesting stream properties](#RequestingStreamProperties).
```

Comment:
```suggestion
follows the rules for [requesting stream properties](#RequestingStreamProperties).
```

Reply:
I wanted to avoid using the word "follow" twice with different meanings in the same sentence.

Reply:
I think it should be neither `obeys` nor `follows` since that would imply that that section is talking about function.
I would say something along the lines of "the response to the GET request is analogous to the rules for ... "